jupyterlab_gcedetails/src/components/action_bar.tsx

```diff
@@ -0,0 +1,54 @@
+import * as csstips from 'csstips';
+import * as React from 'react';
+import { stylesheet, classes } from 'typestyle';
+import { css, COLORS } from 'gcp_jupyterlab_shared';
+
+interface Props {
+  primaryLabel: string;
+  secondaryLabel?: string;
+  onPrimaryClick: () => void;
+  onSecondaryClick?: () => void;
+}
+
+const STYLES = stylesheet({
+  actionBar: {
+    padding: '16px',
+    marginTop: '5px',
+    display: 'block',
+    $nest: {
+      '&>*': {
+        marginLeft: '16px',
+      },
+    },
+    ...csstips.horizontal,
+    ...csstips.endJustified,
+  },
+  primaryButton: {
+    backgroundColor: COLORS.blue,
+    color: COLORS.white,
+  },
+});
+
+/** Funtional Component for defining an action bar with buttons. */
+export function ActionBar(props: Props) {
```

Comment:
The shared package has an ActionBar component as well. I would prefer that we parameterize it more to suit our purposes if possible, but I'm OK if we do so in a later PR.

It currently does support adding additional buttons, but it may be that we need more fine-grained control of the styling.

Reply:
Yeah we were originally using this action bar component from the shared package but it's styling was recently updated which caused the rendering issue shown in the first picture. But I can also look into parameterizing the styling for the shared action bar to resolve that issue